casadi/core/sx_function.hpp

```diff
@@ -33,7 +33,7 @@
 namespace casadi {
   /** \brief  An atomic operation for the SXElem virtual machine */
   struct ScalarAtomic {
-    int op;     /// Operator index
+    Operation op;     /// Operator index
```

Comment:
@jaeandersson your thoughts?

Reply:
Please note, that the type of `Operation` is controlled by the `enum`, see: `casadi/core/calculus.hpp:60`
`enum class Operation : unsigned char {`
So, in this PR it is set to `unsigned char`. This can be changed.